cwl_runner/README.md

```diff
@@ -0,0 +1,116 @@
+# Run a CWL workflow
+
+## This example demonstrates running a multi-stage workflow on Google Cloud Platform
+
+* The workflow is launched with a bash script that calls the [Google Cloud SDK](https://cloud.google.com/sdk)
+* The workflow is defined using the [Common Workflow Language (CWL)](http://www.commonwl.org)
+* The workflow stages are orchestrated using [cwltool](https://github.com/common-workflow-language/cwltool/tree/master/cwltool)
+
+Here's what the bash script does to run a CWL workflow:
```

Comment:
There's a "tense" disconnect between the intro here and the steps.
As is, the first bullet should be "Create*s* a disk".
To just update the intro and not the bullets, perhaps make it:

To run a CWL workflow, cwl_runner.sh will: